.gitignore

```diff
@@ -57,3 +57,9 @@ README.html
 
 # ruff
 .ruff_cache
+
+# for text complexity feature:
+env/
+.DS_Store
```

Comment:
This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).